Loaded system config

NielsKorschinsky opened the pull request:
I split up all rest-api configs into loaded and default systems. Also i moved all important settings into one place at the sppmon.py file, as well as giving a small disclaimer. 

I've labled the argument `--minimumLogs` as depricated and introduced the new arg `--loadedSystems`. This new arg-group should make the new settings more understandable. 
I'd like to remove `--minimumLogs` within the next version (or the release). For now, both args do exactly the same to not break running systems.

@baslerj Please test it in your own enviroment. I think I've already set up appropriate settings, but you might need to finetune some settings. Those changes might be a good recommendation for other loaded systems.

baslerj commented:
@NielsKorschinsky I am currently testing and reviewing the changes.  I have one question regarding the comments in sppmon.py that describe manual changes to make in increasing order of risk.  Where does the use of --loaded_systems fit into that list?  Does it automate some of those changes, should the new option be used before attempting any of the manual changes, or does it come after in the category of not recording all data?

NielsKorschinsky commented:
Hello @baslerj 
Thanks for the hint, I've modified the description but maybe you could also improve it.

Using `--loaded_systems` directly is like the easy way, if you dont want to think a lot.
If you do want to make manual finetuning and changes, i've wanted to give some assistance by adding a priority list. 
The steps later in the list are more radical or have a bigger drawback, due which i want to have the user change other variables first.
`--loaded_systems` would fit in at the step of switching to `SUMMARY`-logs only. 

Now I've split this into critical and minor spikes with recommendations for each situation. I'm still not sure what is the best way for this describtion, I'm only sure that we need some kind of.

baslerj commented:
Recent updates look good.  I have successfully tested the updates in our large SPPmon production environment.
